integration-libs/cdc/core/auth/services/user-authentication/cdc-login.service.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { TestBed, waitForAsync } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { RouterTestingModule } from '@angular/router/testing';
+import {
+    AuthService,
+    GlobalMessageService,
+    I18nTestingModule,
+    WindowRef
+} from '@spartacus/core';
+import { LoginFormComponentService } from 'feature-libs/user/account/components/login-form/login-form-component.service';
+import { CdcJsService } from 'integration-libs/cdc/root';
+import { FormErrorsModule } from 'projects/storefrontlib/shared';
```

Comment:
Change import of FormErrorsModule to Storefront

Reply:
noted